angstudy/src/app/components/shop-card/shop-card.component.html

```diff
@@ -1,17 +1,23 @@
-<mat-card class="example-card" (click)="putOnBusket($event)">
-    <!-- <mat-card-header>
-      <div mat-card-avatar class="example-header-image"></div>
-      <mat-card-title>Shiba Inu</mat-card-title>
-      <mat-card-subtitle>Dog Breed</mat-card-subtitle>
-    </mat-card-header> -->
-    <img mat-card-image src={{image}} alt="Goods images">
+<mat-card class="example-card" (click)="putOnBusket()">
+    <!-- тут какая то ошибка со стилями, не пойму как исправить. отображается тока правая стрелка в карусели -->
+    <ng-container *ngIf="product.images as Images">
```

Comment:
```suggestion
    <ng-container *ngIf="product.images as images">
```
images - переменная в шаблоне, следовательно должна начинаться с маленькой буквы